DXRMapFixups/DeusEx/Classes/DXRFixupM06.uc

```diff
@@ -953,8 +958,21 @@ function AnyEntryMapFixes()
             m.bLocked = false;
             m.bHighlight = true;
         }
+
         HandleJohnSmithDeath();
         FixMaggieMoveSpeed();
+
+        if (dxr.flags.settings.starting_map >= 68 || dxr.flagbase.GetBool('M06_HONGKONG_MJ12LAB_Randomized')) {
+            foreach AllActors(class'#var(prefix)MaggieChow', maggie) {
+                maggie.LeaveWorld();
+                break;
+            }
+            foreach AllActors(class'ScriptedPawn', p, 'MaggieMaid') {
```

Comment:
Why remove the maid?  She's intentionally supposed to be hostile if you arrive later on

Reply:
There's a conversation for if you return after going to the police vault where she turns hostile, which is the only one I've ever encountered. You specifically talk about it. But it actually looks like there's also a conversation for if you meet her and Maggie is already dead (she doesn't turn hostile). Could repurpose that conversation.